.github/workflows/integration.yml

```
@@ -125,3 +125,43 @@ jobs:
       - name: ${{ matrix.crate }}
         working-directory: ./${{ matrix.crate }}
         run: cargo ${{ matrix.args }}
+
+  msrv:
+    name: Minimum Supported Rust Version
+    if: github.repository == 'aws/aws-lc-rs'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-12, macos-13-xlarge, windows-latest ]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+          lfs: true
+
+      - name: Install NASM on Windows
+        if: runner.os == 'Windows'
+        uses: ilammy/setup-nasm@v1
+
+      - name: Setup Go >=v1.18
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.18'
+
+      - name: Install ninja-build tool
+        if: runner.os == 'Windows'
+        uses: seanmiddleditch/gha-setup-ninja@v4
+
+      - uses: dtolnay/rust-toolchain@stable
+        id: toolchain
+
+      - name: Set Rust toolchain override
+        run: rustup override set ${{ steps.toolchain.outputs.name }}
+
+      - name: Install cargo-msrv
+        run: cargo install cargo-msrv
+
+      - name: Verify msrv
+        working-directory: ./aws-lc-rs
+        run: cargo msrv --log-level debug --log-target stdout verify -- cargo check --features bindgen
```

Any reason this cant be `build --all-features` so its a bit more comprehensive?

```suggestion
        run: cargo msrv --log-level debug --log-target stdout verify -- cargo build --all-features
```